packages/components/src/MultiSelect/MultiSelect.container.test.js

```diff
@@ -3,9 +3,41 @@ import { shallow } from 'enzyme';
 
 import Container from './MultiSelect.container';
 
+function t(id, options) {
+	return options.defaultValue;
+}
+
 describe('Container MultiSelect', () => {
-	it('should render MultiSelect with props', () => {
-		const wrapper = shallow(<Container />);
-		expect(wrapper.props()).toMatchSnapshot();
+	it('should render input with props', () => {
+		const wrapper = shallow(<Container.WrappedComponent id="multiselect" />);
+		expect(wrapper.find('input').props()).toMatchSnapshot();
+	});
+	it('getOptions should eleminate duplicate from parent', () => {
```

Comment:
Avoid testing internals, that makes the tests not robust, as it fails in case of refacto, even if the component behavior hasn't changed.
You should: 
1- mount the component
2- interact with it
3- assert what is rendered

Reply:
I didn t succeed to mount the virtualized list